libs/menu/src/DropdownMenu.tsx

```diff
@@ -252,17 +257,27 @@ function DropdownMenuContainer({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [childrenContainerRef.current]);
 
+  const getHeight = useCallback(() => {
```

Comment:
Why not useMemo instead of useCallback? I don't see a reason that this needs to be callable